src/composables/useNumbers.ts

```diff
@@ -62,5 +90,67 @@ export default function useNumbers() {
     return tokenAmount.times(price).toString();
   }
 
-  return { fNum, toFiat };
+  function fNum2(
+    number: number | string,
+    options: FNumOptions | undefined = {}
+  ): string {
+    if (typeof number === 'string') {
+      number = number || 0;
+      number = new BigNumber(number).toNumber();
+    }
```

Comment:
could you leave a comment as to why this is needed

Reply:
I thought using `parseFloat` or `Number` may cause issues with floating point precision. I added a test and it seems this isn't the case so made it just use `Number` instead now rather than this.